fhir-examples/pom.xml

```diff
@@ -17,7 +17,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-checkstyle-plugin</artifactId>
-                <version>4.0.0-SNAPSHOT</version>
+                <version>3.1.0</version>
```

Comment:
why the version change?

Reply:
4.0.0-SNAPSHOT looked like an error on the plugin.

Reply:
discussed offline